packages/page-accounts/src/modals/Qr.tsx

```diff
@@ -138,8 +143,12 @@ function QrModal ({ className = '', onClose, onStatusChange }: Props): React.Rea
           : (
             <Modal.Columns hint={t<string>('Provide the account QR from the module/external application for scanning. Once detected as valid, you will be taken to the next step to add the account to your list.')}>
               <div className='qr-wrapper'>
-                <QrScanAddress onScan={_onScan} />
+                <QrScanAddress isEthereum={isEthereum}
+                  onScan={_onScan} />
               </div>
+              {
+                warning && <MarkWarning>{warning}</MarkWarning>
+              }
```

Comment:
```suggestion
              {warning && <MarkWarning content={warning} />}
```

The component is a bit "funny" in this regard. (Still not sure if it is a good idea or a bad one atm...)